tests/stats_monitor/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+rm -rf output
+hindsight_cli hindsight.cfg 7
```

Comment:
You cannot use the CLI here as it doesn't actively update the stats after the input exits (the extra kill -2 code is to allow us to use hindsight proper and self shutdown)